assets/src/design-system/components/toggle/index.js

```diff
@@ -87,9 +90,10 @@ const StyledCheckmark = styled(Checkmark)(
 
 const ToggleContainer = styled.div(
   ({ theme }) => css`
+    box-sizing: border-box;
     position: relative;
-    height: ${TOGGLE_HEIGHT}px;
-    width: ${TOGGLE_WIDTH}px;
+    height: ${TOGGLE_HEIGHT - BORDER_WIDTH * 2}px;
+    width: ${TOGGLE_WIDTH - BORDER_WIDTH * 2}px;
```

Comment:
The design system uses the default `box-sizing: content-box` but the dashboard and editor uses `box-sizing: border-box;`

I think it would be prudent to make everybody use the same sizing system 😅. That way things aren't just barely off between them!

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/111001584-4e051180-8341-11eb-8df7-4dc81abca470.png)|![image](https://user-images.githubusercontent.com/22185279/111001559-42194f80-8341-11eb-8c32-500b749b650e.png)|

Reply:
Yea I normally exclusively use `border-box`. I'm curious what the use case was for the design system to use `content-box` ?

Reply:
I can go through it later to update to use `border-box`. I don't think there's a specific use case that limits us to using `context-box`.

Reply:
we should use border-box, there's no reason for context-box, that's just the default.  I made a tech debt ticket, @samwhale if you feel like going through it later: https://github.com/google/web-stories-wp/issues/6775

Reply:
thank you!!

Reply:
Moved these changes to https://github.com/google/web-stories-wp/pull/6779 which addresses issue #6775